braindecode/models/config.py

```diff
@@ -0,0 +1,217 @@
+try:
+    import pydantic
+except ImportError:
+    raise ImportError(
+        "pydantic is not installed. Install with: pip install braindecode[pydantic]"
+    )
+import functools
+import operator
+from collections.abc import Callable
+from inspect import signature
+from types import UnionType
+from typing import Annotated, Any, Literal, Union, get_args, get_origin
+
+import numpy as np
+from numpydantic import NDArray, Shape
+from typing_extensions import TypedDict
+
+from braindecode.models.base import EEGModuleMixin
+from braindecode.models.util import SigArgName, models_dict, models_mandatory_parameters
+
+
+class ChsInfoType(TypedDict, total=False):
```

Comment:
We have more fields in the chs info:

https://github.com/mne-tools/mne-python/blob/09905e5745b2e5f3c81b442e2ce36558af181c54/mne/_fiff/tests/test_meas_info.py#L928

Reply:
This is just the `"chs"` key of info. I took all the fields described in the [doc](https://mne.tools/stable/generated/mne.Info.html) (see Notes section)